anchore_engine/clients/grype_wrapper.py

```diff
@@ -866,19 +869,10 @@ def query_record_source_counts(self, use_staging: bool = False):
                 # Transform the results along with the last_synced timestamp for each result
                 output = []
                 for result in results:
```

Comment:
(nit) unpacking the tuple here makes for easier reading. It would also raise exceptions sooner if the expected number of elements are not available
```
for group, count in results:
```